packages/react/package.json

```diff
@@ -38,9 +38,8 @@
     "@types/chai-string": "^1.4.5",
     "@types/mocha": "^10.0.10",
     "@types/node": "22.12.0",
-    "@types/prop-types": "^15.7.14",
-    "@types/react": "^18.2.45",
-    "@types/react-dom": "^18.0.10",
+    "@types/react": "^19.0.10",
```

Comment:
i see different versions for types. Is there a reason for this?
Can we keep it consistent?